assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -275,6 +275,8 @@ const MediaElement = ({
         width={width}
         height={height}
         preload="none"
+        role="application"
```

Comment:
That's a big role for a video... I don't think we should do that.

> The `application` role indicates to assistive technologies that an element and all of its children should be treated similar to a desktop application, and no traditional HTML interpretation techniques should be used. This role should only be used to define very dynamic and desktop-like web applications.

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/application_role

Reply:
That's right, but we want to use getByRole if possible and it's the only role applicable to video tag (and I didn't find any argument against it), so we need to find something equally good for a replacement. Anything better than testid? WDYT @barklund ?

Reply:
I think the above is a pretty big argument against it 🙂 

The `figure` role would be more appropriate, but still wrong, as `figure` usually means video/image/audio + caption, plus a `video` is already quite semantic.

I'd stress your "getByRole if possible" point and say that if there's no suitable role, use a test ID

Reply:
The only allowed role for video is `application` or no role. I did suspect that `application` might be dangerous though, but it was my suggestion. But could we perhaps put it in a `<figure>` and still access the `<video>` tag with simple RTL selectors - `getByRole` or `getByLabelText`?

Reply:
I have [summoned the gods of RTL](https://twitter.com/barklund/status/1271461392575598593)...

Reply:
Let's try just `byLabel` here. I'll fix and see if it works.

Reply:
Remember to remove the role here too!